app/main.py

```diff
@@ -1,2 +1,111 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.length_hash = 8
+        self.length_dict = 0
+        self.capacity = [[] for _ in range(self.length_hash)]
+
+    def __setitem__(self, key, value):
+        if self.length_dict >= self.length_hash * 2 // 3:
+            temp_capacity = self.capacity.copy()
+            self.length_hash *= 2
+            self.capacity = [[] for _ in range(self.length_hash)]
+            for item in temp_capacity:
```

Comment:
It's actually the logic of setitem method. So call it from the new `resize` method you will create